config/db/seed.js

```diff
@@ -315,22 +443,61 @@ async function dropDb() {
 
 async function createParticipations() {
   const users = await User.find();
-  await Participation.create(
-    users.map(user => {
-      challenge: currentChallenge, user;
-    })
-  );
+  users.map(user => {
+    Participation.create({
+      challenge: currentChallenge,
+      user
+    });
+  });
+}
+
+async function createUsers() {
+  const families = await Family.find();
+  const users = require("./json/users.json");
+  users.map(user => {
+    User.create({
+      family: families[Math.floor(Math.random() * families.length)],
+      email: user.email,
+      name: {
+        first: user.name.first,
```

Comment:
Why are you redefining the scaffolding here? The seed data should do this. You should only need to call User.create({user}) and it should just work.

Reply:
I wasn't able to simply just pass the user object since I stored the user info in json and needed to set the family attribute to one of the family objects. With the rewrite this will however be resolved.

Reply:
Removed scaffolding. resolving.